core/modules/wmgr/python/config.py

```diff
@@ -99,11 +99,11 @@ def dbConn(self):
         kwargs = {}
         if self.dbHost: kwargs['host'] = self.dbHost
         if self.dbPort: kwargs['port'] = self.dbPort
-        if self.dbSocket: kwargs['unix_socket'] = self.dbSocket
+        if self.dbSocket: kwargs['query'] = {"unix_socket": self.dbSocket}
         if self.dbUser: kwargs['user'] = self.dbUser
         _log.debug('creating new connection %s', kwargs)
         if self.dbPasswd: kwargs['passwd'] = self.dbPasswd
-        inst = db.Db(**kwargs)
+        inst = db.Db().getEngineFromArgs(**kwargs)
```

Comment:
It may be better to rename method from `dbConn()` to `dbEngine()` as it returns engine instance now. Same for `privDbConn()`